app/views/users/show/pr/_icon.html.erb

```diff
@@ -13,6 +13,11 @@
     <div class="unaccepted-box">
     </div>
   </div>
+<% elsif pr_type == 0 && pr.eligible? %>
```

Comment:
Nitpick, but I think it'd be clearer if we replace `pr_type` with something like `is_bonus` and have it be a boolean?